content/fr/integrations/amazon_event_bridge.md

```diff
@@ -9,6 +9,7 @@ doc_link: 'https://docs.datadoghq.com/integrations/amazon_event_bridge/'
 draft: false
 git_integration_title: amazon_event_bridge
 has_logo: true
+integration_id: ''
```

Comment:
@davidejones We updated the `integration_ids` recently, is this okay to merge as-is?

Reply:
@sarina-dd yup it should be fine, I checked and these are ok to be empty if the source was empty. I also merged in master into this branch and checked and it seems good here 👍